packages/cli-tree/src/completion.js

```diff
@@ -0,0 +1,84 @@
+/* eslint no-console: off */
+const tabtab = require('tabtab')
+const get = require('lodash/get')
+const toPairs = require('lodash/toPairs')
+const _ = require('lodash')
+
+module.exports = {
+  getCompletionSetupCommands,
+  completionHandler: async (commands = {}) => {
```

Comment:
Can we have the completion handler at module level, like the other functions that are exported ?

Reply:
I am not sure to understand. the completionHandler is available at the module level. You would prefer to have the completionHandler code in index.js ?

Reply:
I meant transforming into:

```

module.exports = {
  getCompletionSetupCommands,
  completionHandler,
  ...
}

const completionHandler = async (commands = {}) => {
  ...
}
```